editor/editor_interface.cpp

```diff
@@ -512,6 +513,38 @@ void EditorInterface::popup_quick_open(const Callable &p_callback, const TypedAr
 	quick_open->popup_dialog(base_types, callable_mp(this, &EditorInterface::_quick_open).bind(p_callback));
 }
 
+void EditorInterface::popup_create_dialog(const Callable &p_callback, const StringName &p_base_type, const TypedArray<StringName> &p_custom_type_blacklist, const Dictionary &p_custom_suffix) {
+	if (!create_dialog) {
+		create_dialog = memnew(CreateDialog);
+		get_base_control()->add_child(create_dialog);
+	}
+
+	HashSet<StringName> blacklist;
+	for (int i = 0; i < p_custom_type_blacklist.size(); i++) {
+		blacklist.insert(p_custom_type_blacklist[i]);
+	}
+	create_dialog->set_type_blacklist(blacklist);
+
+	HashMap<StringName, String> suffix_map;
+	for (int j = 0; j < p_custom_suffix.size(); j++) {
+		StringName k = p_custom_suffix.get_key_at_index(j);
+		if (k.is_empty()) {
+			// Skips empty keys and invalid values. The dictionary should be Dictionary[StringName, String],
+			// but due to a bug when using TypedDictionary, the final compile will failed due to being unable to
+			// parse ";" after "typeddictionary::" during the godot-cpp binding.
+			continue;
+		}
+		suffix_map.insert(k, p_custom_suffix[k]);
+	}
+	create_dialog->set_type_suffixes(suffix_map);
+
+	create_dialog->set_base_type(p_base_type.is_empty() ? "Object" : p_base_type);
+	create_dialog->select_base();
+	create_dialog->popup_create(false, true, "", "");
+	create_dialog->connect(SNAME("create"), callable_mp(this, &EditorInterface::_create_dialog_item_selected).bind(false, p_callback));
+	create_dialog->connect(SNAME("canceled"), callable_mp(this, &EditorInterface::_create_dialog_item_selected).bind(true, p_callback));
```

Comment:
Why not use `const Callable` like in other callbacks?

Reply:
I meant this: `const Callable callback = callable_mp(this, &EditorInterface::_create_dialog_item_selected)`.
https://github.com/godotengine/godot/blob/a372214a4a047ccf63d23c4406337b30e55ff60b/editor/editor_interface.cpp#L459-L461
Avoids repeating `callable_mp()`. Note that `callback` and `p_callback` are two different Callables (maybe name could be better).